src/app/home/home.component.html

```diff
@@ -0,0 +1,91 @@
+<app-header></app-header>
```

Comment:
I missed before, that you add header for each component
you can do it like here in app component:
so header will be everywhere
![image](https://github.com/user-attachments/assets/c3956619-15d5-4d8f-b6db-1e6d46f8ff0c)

Reply:
done